Cargo.toml

```diff
@@ -22,4 +22,4 @@
     wasmtime-wasi               = "0.33"
     wasi-common                 = "0.33"
     wasi-cap-std-sync           = "0.33"
-    wit-bindgen-wasmtime        = { git = "https://github.com/bytecodealliance/wit-bindgen", rev = "2e654dc82b7f9331719ba617a36ed5967b2aecb0" }
+    wit-bindgen-wasmtime        = { git = "https://github.com/bytecodealliance/wit-bindgen", rev = "8fdf702982ab141c63d5b1885994306c77dbc41d" }
```

Comment:
First, you are only bumping the revision of `wit-bindgen` here, whereas this crate is imported in the top level `Cargo.toml` — so the first thing I'd do is either bump all, or none.